Documentation/cluster-crd.md

```diff
@@ -201,3 +222,34 @@ spec:
       nodeAffinity:
       tolerations:
 ```
+
+### Resource requests/Limits
+
+To control how many resources the rook components can request/use, you can set requests and limits in Kubernetes for them.
+The example under `resources` in `spec` sets requests on a cluster level. You can override
+these requests/limits for OSDs per node when not using `useAllNodes: false` in the `node` item in the `nodes` list.
```

Comment:
What if you only mention the special behavior for the OSDs in the section on the storage nodes? Seems like it is confusing when you mention it here where the focus is the overall cluster level.